From f9921c6a782e18c0efd0d70f782921a706e9ebd0 Mon Sep 17 00:00:00 2001
From: "Peter A. Bigot" <pab@pabigot.com>
Date: Fri, 16 Aug 2013 12:38:09 -0500
Subject: [PATCH 32/32] OMAP3: overo: increase linux partition to 8 MiB

Linux kernel at version 3.5 is about 3.5 MiB; test kernels for 3.10 exceed 4
MiB.  Prepare for future upgrades by increasing the NAND partition now.

Upstream-Status: Pending

Signed-off-by: Peter A. Bigot <pab@pabigot.com>
---
 arch/arm/mach-omap2/board-overo.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-omap2/board-overo.c b/arch/arm/mach-omap2/board-overo.c
index 055e59e..460ae96 100644
--- a/arch/arm/mach-omap2/board-overo.c
+++ b/arch/arm/mach-omap2/board-overo.c
@@ -370,11 +370,11 @@ static struct mtd_partition overo_nand_partitions[] = {
 	{
 		.name           = "linux",
 		.offset         = MTDPART_OFS_APPEND,	/* Offset = 0x280000 */
-		.size           = 32 * NAND_BLOCK_SIZE,
+		.size           = 64 * NAND_BLOCK_SIZE,
 	},
 	{
 		.name           = "rootfs",
-		.offset         = MTDPART_OFS_APPEND,	/* Offset = 0x680000 */
+		.offset         = MTDPART_OFS_APPEND,	/* Offset = 0xa80000 */
 		.size           = MTDPART_SIZ_FULL,
 	},
 };
--
1.7.9.5

